<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>research</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 64px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h2 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h3 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h4 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h5 {
  padding-top: 69px;
  margin-top: -69px;
}
.section h6 {
  padding-top: 69px;
  margin-top: -69px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">andrew j helmstetter</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="research.html">research</a>
</li>
<li>
  <a href="cv.html">cv</a>
</li>
<li>
  <a href="publications.html">publications</a>
</li>
<li>
  <a href="photography.html">photography</a>
</li>
<li>
  <a href="blog.html">blog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="mailto:andrew.j.helmstetter@gmail.com">
    <span class="fa fa-envelope fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://scholar.google.com/citations?user=Ye-lhkIAAAAJ&amp;hl=en">
    <span class="fa fa-graduation-cap fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://twitter.com/ajhelmstetter">
    <span class="fa fa-twitter fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="http://github.com/ajhelmstetter">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="https://www.instagram.com/unchatquipeche">
    <span class="fa fa-instagram fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">research</h1>

</div>


<p>I use a variety of approaches and study groups to understand the evolutionary origins of diversity. My research typically revolves around phylogenetics, population genetics and genomics but also delves into plant health, modelling species distributions and examining patterns of growth and development.</p>
<table id="myTable" style="padding-top:16px padding-bottom:16px">
<tr style="padding-top:30px padding-bottom:30px">
<td>
<div id="from-reproductive-strategies-to-species-diversity-how-evolution-of-breeding-systems-and-associated-traits-shape-plant-biodiversity" class="section level4">
<h4>From reproductive strategies to species diversity: how evolution of breeding systems and associated traits shape plant biodiversity</h4>
<p><font size="2"></p>
<p><em>with <a href="https://scholar.google.fr/citations?user=Ol-P2LQAAAAJ&amp;hl=en">Sylvain Glemin</a>, <a href="https://lbbe.univ-lyon1.fr/-Kafer-Jos-.html?lang=fr">Jos Kafer</a> and <a href="https://www.fondationbiodiversite.fr/en/the-frb-in-action/programs-and-projects/le-cesab/divers/">DiveRS</a></em></p>
<p>Angiosperms (flowering plants) present an exceptional diversity of breeding systems, with variation both in gender distribution within and among individuals (from hermaphroditism to separate sexes) and in mating patterns (from strict outcrossing to predominant selfing). Breeding systems have been shown to affect species diversification and to be associated with other life-history and ecological traits. Consequently, breeding systems could be an important determinant of the observed species diversity in flowering plants. The <strong>DiveRS</strong> project aims at exploring the hypothesis that the combination of traits associated with breeding systems could be an important key to understanding evolutionary success and plant species diversity.</p>
<p></font></p>
</td>
<td align="center" vertical-align="middle">
<p><img src="images/divers.png" style="width:85.0%" /></p>
</td>
</tr>
<tr style="padding-top:30px padding-bottom:30px">
<td>
</div>
<div id="evolutionary-dynamics-of-central-african-rain-forests" class="section level4">
<h4>Evolutionary dynamics of central African rain forests</h4>
<p><font size="2"></p>
<p><em>with <a href="http://www.couvreurlab.org/">Thomas Couvreur</a></em></p>
<p>Tropical rain forests (TRFs) are hyper-diverse ecosystems and ongoing climate change will have an extraordinary impact on their biodiversity. Studying how past climate change, for example the Pleistocene climatic cycles, impacted genetic diversity provides insights into how species reacted to such change. Central Africa contains the second largest expanse of rain forest worldwide and is therefore an ideal system to study how climate change affects diversity. It is thought that most species responded similarly to past climate change, principally through shared glacial refugia. However, we have no temporal scale for these responses, which means we are missing a key piece of the puzzle. To address this shortcoming we used cutting-edge methods to generate a large and comprehensive genomic dataset of over 750 individuals, unparalleled for African TRFs. We use this to estimate the evolutionary and demographic history of seven co-distributed TRF plant species.</p>
</font>
</td>
<td align="center" vertical-align="middle">
<p><img src="images/afrodyn.png" style="width:85.0%" /></p>
</td>
</tr>
<tr style="padding-top:30px padding-bottom:30px">
<td>
</div>
<div id="development-of-primary-genomic-resources-for-securing-sustainable-hazelnut-production-in-turkey" class="section level4">
<h4>Development of primary genomic resources for securing sustainable hazelnut production in Turkey</h4>
<p><font size="2"></p>
<p><em>with <a href="http://richardbuggs.com/">Richard Buggs</a> and <a href="http://myweb.sabanciuniv.edu/slucas/">Stuart Lucas</a></em></p>
<p>Hazelnut ( <em>Corylus avellana</em> L. ) is consumed worldwide and is of critical economic importance to the rural communities of Turkey’s northern coast. A new disease outbreak has drastically decreased yields across Turkey and climate change is emerging as a new threat to cultivation. Our study is the first to provide a genomic perspective on diversity in this vulnerable crop, which will prove valuable for future breeding efforts. We also provide a new p0hylogenetic hypothesis, shoiwng how the cosmopolitan genus <em>Corylus</em> diversified worldwide. Such research into perennial crops like hazelnut can help to improve farmer livelihoods and ensure the sustainability of crop production in a changing world.</p>
</font>
</td>
<td align="center" vertical-align="middle">
<p><img src="images/hazel.png" style="width:85.0%" /></p>
</td>
</tr>
<tr style="padding-top:30px padding-bottom:30px">
<td>
</div>
<div id="demography-and-phylogeny-of-madagascan-flora" class="section level4">
<h4>Demography and Phylogeny of Madagascan Flora</h4>
<p><font size="2"></p>
<p><em>with <a href="http://labadopulos.co.uk/">Alex Papadopulos</a></em></p>
<p>Current IUCN conservation assessment methods are reliant on observed declines in species population and range sizes over the last one hundred years, but for the majority of species this information is not available. We used a population genetic appraoch to reveal historical demographic decline in the rare endemic flora of Madagascar. These results show that it is possible to predict extinction risk from demographic patterns inferred from genetic data and that destructive human influence is likely to have resulted in the very high frequency of narrow endemics present on the island. Our approach will act as an important tool for rapidly assessing the threatened status of poorly known species in need of further study and conservation, particularly for tropical flora and fauna.</p>
</font>
</td>
<td align="center" vertical-align="middle">
<p><img src="images/madagascar.png" style="width:85.0%" /></p>
</td>
</tr>
<tr style="padding-top:30px padding-bottom:30px">
<td>
</div>
<div id="real-time-genomics-for-species-identification" class="section level4">
<h4>Real-time genomics for species identification</h4>
<p><font size="2"></p>
<p><em>with <a href="http://www.lonelyjoeparker.com/">Joe Parker</a> and <a href="http://labadopulos.co.uk/">Alex Papadopulos</a></em></p>
<p>Recently, portable, real-time, nanopore sequencing (RTnS) offers opportunities to rapidly collect and analyse genomic data anywhere. However, generation of datasets from large, complex genomes has been constrained to laboratories. The portability and long DNA sequences of RTnS offer great potential for field-based species identification, but the feasibility and accuracy of these technologies for this purpose have not been assessed. We show that a field-based RTnS analysis of closely-related plant species ( <em>Arabidopsis</em> spp.) has many advantages over laboratory-based high-throughput sequencing (HTS) methods for species level identification and phylogenomics. Samples were collected and sequenced in a single day by RTnS using a portable, <em>al fresco</em> laboratory. Our analyses demonstrate that correctly identifying unknown reads from matches to a reference database with RTnS reads enables rapid and confident species identification. We have extended this technology and used it in practice for outreach events to teach children the wonders of real-time DNA sequencing.</p>
</font>
</td>
<td align="center" vertical-align="middle">
<p><img src="images/minion.png" style="width:85.0%" /></p>
</td>
</tr>
<tr style="padding-top:30px padding-bottom:30px">
<td>
</div>
<div id="adaptation-and-speciation-genetics-in-silene" class="section level4">
<h4>Adaptation and Speciation Genetics in <em>Silene</em></h4>
<p><font size="2"></p>
<p><em>with <a href="http://labadopulos.co.uk/">Alex Papadopulos</a></em></p>
<p>The sea campion, <em>Silene uniflora</em> is a small flowering plant that is found on coastlines throughout the UK. According to folklore, sea campion should never be picked for fear of tempting death. In several locations throughout the UK, this species has colonised and adapted to the highly toxic soils produced by mining in the last 200 years. We are using a combination of population genomic, gene expression and ecological experiments to address questions about the repeatability of evolution as a result of human activities.</p>
</font>
</td>
<td align="center" vertical-align="middle">
<p><img src="images/silene.png" style="width:85.0%" /></p>
</td>
</tr>
<tr style="padding-top:30px padding-bottom:30px">
<td>
</div>
<div id="diversification-in-cyprinodontiformes" class="section level4">
<h4>Diversification in Cyprinodontiformes</h4>
<p><font size="2"></p>
<p><em>with <a href="http://tomvandooren.eu/">Tom Van Dooren</a>, <a href="http://labadopulos.co.uk/">Alex Papadopulos</a> and <a href="https://www.javierigea.com/">Javi Igea</a></em></p>
<p>The genus <em>Austrolebias</em> (order: Cyprinodontiformes) consists of over 40 species of annual killifish found in seasonal ponds distributed across the grasslands, wetlands and savannah of eastern South America. My research aims to uncover why, when and how phenotypic differences among <em>Austrolebias</em> differences evolved in its ephemeral pond habitat. I also investigate the structure of the <em>Austrolebias</em> genome using high-throughput sequencing approaches to construct a genetic map from hybrid individuals. I also look at macroevolutionary processes in Cyprinodontiformes, an order of approximately 1,200 ray-finned fish species found primarily in Africa and the Americas. Many of these species are popular in the aquarium trade including guppies, mollies and killifish. Living in a wide range of habitats, they have also evolved different and peculiar reproductive life history strategies such as annualism or viviparity. My research aims to understand the factors that shape diversity in this order.</p>
</font>
</td>
<td align="center" vertical-align="middle">
<p><img src="images/austrolebias.png" style="width:85.0%" /></p>
</td>
</tr>
</table>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
